.drone.yml

```diff
@@ -51,9 +32,51 @@ trigger:
     exclude:
     - promote
 
+---
+kind: pipeline
+name: publish-linux
+type: docker
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: publish-linux
+  image: rancher/hardened-build-base:v1.16.4b7
+  commands: 
+  - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+  - DRONE_TAG=${DRONE_TAG} make publish-linux-amd64
```

Comment:
This should be the command in the below script

Reply:
fixed